Art/gauravsaxena816/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+      crossorigin="anonymous"
+    />
+    <link rel="shortcut icon" href="ZTM.png" type="image/x-icon" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css"
+    />
+    <title>Animation Nation - ZTM Hacktoberfest</title>
+  </head>
+
+  <body>
+    <div class="header" id="head">
+      <h1>Animation Nation</h1>
+      <h2>A ZTM Challenge for Hacktoberfest</h2>
+    </div>
+
+    <ul class="flex" id="cards">
+      <!-- Content is generated dynamically  -->
+      <!-- Add your card in include.js following the instructions -->
```

Comment:
This is not necessary. You have a separate `loader.html` which seems to contain the code for your animation. Please delete this file as it is just a copy of the main `index.html`.